src/Simplify.elm

```diff
@@ -7814,6 +7826,10 @@ arrayCollection =
         }
     , nameForSize = "length"
     , determineSize = arrayDetermineSize
+    , fromListLiteralRange =
+        \lookupTable expr ->
+            AstHelpers.getSpecificFunctionCall ( [ "Array" ], "fromList" ) lookupTable expr
+                |> Maybe.andThen (\{ firstArg } -> AstHelpers.getListLiteralRange firstArg)
```

Comment:
All looks good!

(I _personally_ dislike record destructuring because of potential future name clashes etc, so I always prefer
```elm
|> Maybe.andThen (\call -> AstHelpers.getListLiteralRange call.firstArg)
```
instead of
```elm
|> Maybe.andThen (\{ firstArg } -> AstHelpers.getListLiteralRange firstArg)
```
but no need to change this now. Should be fine)

Reply:
Sure. Will change it in a following PR.